charts/sn-platform/templates/broker/broker-configmap-functionmesh.yaml

```diff
@@ -21,7 +21,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: "builtin-connectors"
+  name: {{ .Release.Name }}-{{ include "pulsar.builtinConnectorName" . }}
```

Comment:
> Why not using `pulsar.functionMeshBuiltinConnectors` directly?

`pulsar.functionMeshBuiltinConnectors` is used for `builtinConnectorRef` which has two scenarios:
- It can be assigned a config map that is created by our chart as the default
- It can also be assigned with a customized config map that is created by the customer 
This configmap is created by our chart, so I used the a new template to generate the name